api/src/utils/get-default-value.ts

```diff
@@ -45,3 +49,19 @@ function castToBoolean(value: any): boolean {
 
 	return Boolean(value);
 }
+
+function castToObject(value: any): Record<string, any> | any[] {
+	if (typeof value === 'object') return value;
+
+	if (typeof value === 'string') {
+		try {
+			return JSON.parse(value);
+		} catch (err: any) {
+			if (env.NODE_ENV === 'development') {
+				logger.error(err);
+			}
+		}
```

Comment:
We should return the original value still when the json conversion goes wrong.

```suggestion
		} catch (err: any) {
			if (env.NODE_ENV === 'development') {
				logger.error(err);
			}
			
			return value;
		}
```